src/blocks/author-list/components/ListLayoutSelector.js

```
@@ -3,53 +3,55 @@
 import { __ } from '@wordpress/i18n';
 import { Button, ButtonGroup, Flex, FlexItem } from '@wordpress/components';
-import { list, formatListBullets, formatListNumbered } from '@wordpress/icons';
+import { formatListBullets, formatListNumbered } from '@wordpress/icons';
 
 /**
  * Layout selector component for the Author List block.
  *
- * @param value
- * @param onChange
- * @param {Object} props Component props.
+ * @param          value.value
```

ESLint

[error] Missing JSDoc @param "value.value" type.

(jsdoc/require-param-type)